readme.adoc

```diff
@@ -264,11 +264,13 @@ Unzip the https://github.com/jbangdev/jbang/releases/latest[latest binary releas
 
 === Wrapper install
 
-If you would like to have jbang available in a local directory and committed into a source code repository you akin to maven and gradle wrappers you can use the `wrapper` command.
+If you would like to have `jbang` available in a local directory and committed into a source code repository (akin to Maven and Gradle wrappers) you can use the `jbang wrapper` command.
 
 If you have `jbang` already installed you call `jbang wrapper install` in a folder to install a local `jbang`
 that will run out of that directory using `./jbang`.
 
+The `./.jbang` directory which `jbang wrapper install` creates is just a cache which you typically would not commit to a source code repository, so you can e.g. `echo .jbang/ >>.gitignore`.
```

Comment:
this debatable - the `.jbang/jbang.jar` file is currently the only way we know what version of jbang you installed. we don't have a `.jbang/jbang-wrapper.properties` similar to ` .mvn/wrapper/maven-wrapper.properties` which mvn wrapper uses to know which version of jbang you use.

As is today if you don't commit the .jbang folder you will get the latest downloaded ...which is probably just fine - but just saying that the folder isn't 100% a cache as it encodes that "

so we'll wanna find a different way to phrase it.